### ISSUE TITLE
Tweego cannot decode Snowman 3.

### ISSUE THREAD

**Issue by byronpendason** [videlais/snowman#853]
I downloaded the snowman 3 `format.js` from https://github.com/videlais/snowman/blob/7010a39cc16b8eab49e46eb0fe850cb12f900591/dist/format.js using wget into my `~/.storyformats/snowman-3/` using wget. Then I entered `tweego --list-formats` and got the following output: 

```
~ $ tweego --list-formats
warning: format snowman-3: Skipping format; Could not decode story format JSON chunk.

Available formats:
  ID                     Name (Version) [Details]
  --------------------   ------------------------------
  chapbook-1             Chapbook (2.2.0)
  harlowe-1              Harlowe (1.2.4)
  harlowe-2              Harlowe (2.1.0)
  harlowe-3              Harlowe (3.1.0)
  paperthin-1            Paperthin (1.0.0) [proofing]
  snowman-1              Snowman (1.4.0)
  snowman-2              Snowman (2.0.2)
  sugarcube-1            SugarCube (1.0.35)
  sugarcube-2            SugarCube (2.30.0)
```

Wasn't sure this was a tweego issue or a Snowman one until I tried to use format.js in another third party twine app (Tweezel) and got an error (a unhelpful "something went wrong" type of error).

**Comment by byronpendason** [comment]
The `format.json` file I downloaded is the one from the `V3.0` branch.

**Comment by david-donachie** [comment]
Looking at the file source, it is full of webpack build errors, so I don't think it's a valid build

**Comment by videlais** [comment]
@byronpendason Yeah, sorry about that. It's not a valid build unless it is part of Twine or release. The issue is part of the JSON properties, which was based on an older unofficial format used by Harlowe. Tweego, and my own Extwee, have hard-coded rules to ignore certain parts of older Harlowe builds to fix this issue. I have not updated Snowman to use a newer fix nor let TME, the maintainer of Tweego, know Snowman might use the older properties.

@david-donachie Correct. It's not a valid build.

**Comment by byronpendason** [comment]
> @byronpendason Yeah, sorry about that. It's not a valid build unless it is part of Twine or release. The issue is part of the JSON properties, which was based on an older unofficial format used by Harlowe. Tweego, and my own Extwee, have hard-coded rules to ignore certain parts of older Harlowe builds to fix this issue. I have not updated Snowman to use a newer fix nor let TME, the maintainer of Tweego, know Snowman might use the older properties.
> 
> @david-donachie Correct. It's not a valid build.

Gotcha. I was hoping to test out the Snowman v3, but if I can't I can't. Lol.

Any guesstimates on Snowman 3's ETA?

**Comment by videlais** [comment]
@byronpendason I don't have an exact date or window at the moment. My strong hope is to return to Twine-related things over the next few weeks and slowly spin things back up. Hopefully, by the end of the year.
